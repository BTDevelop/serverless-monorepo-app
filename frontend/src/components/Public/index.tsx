import * as React from 'react';
import { Segment } from 'semantic-ui-react';

class Public extends React.Component<any, any> {
  public render() {
    return <Segment> This is a public page </Segment>;
  }
}

export default Public;
