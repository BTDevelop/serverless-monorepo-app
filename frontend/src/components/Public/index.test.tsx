import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Public from './';

describe('Public Component', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Public />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
