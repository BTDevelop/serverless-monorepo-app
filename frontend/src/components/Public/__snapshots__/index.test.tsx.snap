// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Public Component should render correctly 1`] = `
<div
  className="ui segment"
>
  This is a public page
</div>
`;
